test/dshell/extra-files/cpp_header_gen/library.d

```diff
@@ -113,7 +116,8 @@ extern(C++) class VTable
 {
     extern(D) int hidden_1() { return 1; }
     int callable_2() { return 2; }
-    extern(Objective-C) int hidden_3() { return 3; }
+    version (D_ObjectiveC)
+        extern(Objective-C) int hidden_3() @selector("hidden_3") { return 3; }
```

Comment:
You could change the linkage to `extern(D)` instead. The test only verifies that the header generator emits functions with appropriate linkage (and doesn't screw with the vtable layout).

Reply:
Well, it was `Objective-C` from the beginning and I'm lazy 😃.